components/common/search/SelectedTags.tsx

```diff
@@ -0,0 +1,55 @@
+import { Tag } from '../Tags';
+
+interface SelectedTagsProps {
+  initTags: string[];
```

Comment:
```suggestion
  tags: string[];
```

내부에 state가 따로 없고 받은대로 보여주는거니까 init은 빼고 좋을듯?